exp-models/addon/models/experiment.js

```diff
@@ -74,5 +83,27 @@ export default DS.Model.extend(JamModel, {
             id: 'experimenter.' + this.get('sessionCollectionId')
         });
         collection.save();
-    }.on('didCreate')
+    }.on('didCreate'),
+
+    setThumbnail(raw) {
+        var self = this;
+
+        var thumbnailId = this.get('thumbnailId');
+        if (thumbnailId) {
+            return this.get('store').findRecord('thumbnail', thumbnailId)
```

Comment:
Can you reuse the `thumbnail` computed property to DRY this code?
https://github.com/CenterForOpenScience/exp-addons/pull/23/files#diff-2d6e780c9343997d7d3e8bcbe18f507fR28